packages/components/index.d.ts

```diff
@@ -2,3 +2,30 @@ declare module '*.scss' {
   const content: Record<string, string>;
   export default content;
 }
+
+declare module 'nebenan-eventproxy' {
+  function eventproxy<Type extends keyof HTMLElementEventMap>(
+    event: Type,
+    callback: () => void
+  ): () => void;
+
+  export default eventproxy;
+}
+
+declare module 'nebenan-react-hocs/lib/use_stable_callback' {
+  function useStableCallback(
+    callback: () => void
+  ) : () => void;
```

Comment:
https://github.com/good-hood-gmbh/nebenan-frontend/blob/a76a106715a3ade3050cc25bfe93d1092d7ee93f/index.d.ts#L161

Reply:
In the linked code <T> is a generic type? Shouldnt it say somehow that callback a function?

Reply:
Ah, you're right! You could extend the generic `T` to something 'callable' (don't know how to do it myself) 

I think main point of the declaration is that whatever callback you pass in, type of returned callback is the same